src/room/GroupCallInspector.tsx

```diff
@@ -427,16 +488,26 @@ function useGroupCallState(client, groupCall, pollCallStats) {
 
   return state;
 }
-
-export function GroupCallInspector({ client, groupCall, show }) {
+interface GroupCallInspectorProps {
+  client: MatrixClient;
+  groupCall: GroupCall;
+  show: boolean;
+}
+export function GroupCallInspector({
+  client,
+  groupCall,
+  show,
+}: GroupCallInspectorProps) {
   const [currentTab, setCurrentTab] = useState("sequence-diagrams");
-  const [selectedUserId, setSelectedUserId] = useState();
+  const [selectedUserId, setSelectedUserId] = useState<string>();
   const state = useGroupCallState(client, groupCall, show);
 
-  const [_, setState] = useContext(InspectorContext);
+  // Review questions: was this supposed to be setContext?
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
setState probably wants to be setContext here, yeah

Reply:
okay I think I didnt get it. Our context is a `[string, React.Dispatch<React.SetStateAction<State>>]` object. so useContext returns our state variable...